astropy/units/tests/test_structured.py

```diff
@@ -10,7 +10,7 @@
 from numpy.testing import assert_array_equal
 
 from astropy import units as u
-from astropy.tests.helper import check_pickling_recovery, pickle_protocol
+from astropy.tests.helper import check_pickling_recovery, pickle_protocol  # noqa: F401
```

Comment:
Why is this necessary to mark it? Both are in fact used inside the module.

Reply:
Pickle protocol is a pytest fixture, so it's only referenced as a function argument. A flake8 plug-in to detect this would be nice!

Reply:
Ah, yes, makes sense.